### DISCUSSION TITLE
hotspots based on pts per meter

### DISCUSSION THREAD

**Discussion by jGaboardi** [pysal/spaghetti#547] [Ideas] [answered]
Raised by @anitagraser in the [Gitter channel](https://gitter.im/pysal/Spaghetti?at=5fad5224f2fd4f60fc6c8872)

> Hi! I've been playing with 1.5.3 to test the fix for #526. While playing around with differently split networks, I observed how drastically the detected hotspots can change. I was wondering if we could compute the hotspots based on pts per meter instead of the absolute point counts. Since graph edges differ in length, longer edges will get more points if points are distributed uniformly.

**Comment by jGaboardi** [comment, marked as answer]
@knaaptime and @weikang9009 Do you have any thoughts on this? My first impression is that it could be as simple as adding in a keyword argument to [`Network.counts_per_link()`](https://github.com/pysal/spaghetti/blob/6560b8be963ed86faadf13f5e7c9c0f1b63ce606/spaghetti/network.py#L1301) (maybe something like `as_ratio`) where the per distance unit count would be returned instead of the absolute count.

**Comment by anitagraser** [comment]
To illustrate the issue: the following plots are based on the same point pattern. 

![image](https://user-images.githubusercontent.com/590385/99077532-30a37f00-25bd-11eb-800b-4d5c95045057.png)
